src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/table/index.js

```diff
@@ -0,0 +1,32 @@
+import convertParagraph from '../paragraph';
```

Comment:
Just needs automated tests :)  If you provide some more fixtures of what input data looks like I can add tests for this one in the other branch (https://github.com/bbc/simorgh/pull/9169)